frontend/src/metabase/entities/collections.js

```diff
@@ -101,15 +101,14 @@ const Collections = createEntity({
     getInitialCollectionId: createSelector(
       [
         state => state.entities.collections,
+
         // these are listed in order of priority
```

Comment:
Any reason why this comment is not above line 103?

Reply:
`state => state.entities.collections` does no collection id lookups. It selects a collection objects dictionary from redux and is used to ensure a user has to write permissions to a suggested collection

Reply:
Thanks.